<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutsDialog</class>
 <widget class="QDialog" name="ShortcutsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcuts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/images/shortcut.png</normaloff>:/icons/images/shortcut.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filterEdit">
     <property name="placeholderText">
      <string>Instruction name filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblShortcuts"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Shortcut</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="ShortcutEdit" name="shortcutEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearShortcut">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutEdit</class>
   <extends>QLineEdit</extends>
   <header>ShortcutEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
